angular/projects/spark-angular/src/lib/components/sprk-highlight-board/sprk-highlight-board.component.ts

```diff
@@ -151,11 +151,21 @@ export class SprkHighlightBoardComponent {
   @Input()
   idString: string;
 
+  ngOnInit() {
+    if (!this.ctaText && this.ctaText2) {
+      console.warn(`
+        sprk-highlight-board: A secondary call-to-action (CTA)
```

Comment:
i think we should standardize on a  console warning text style. I see it done differently here: https://github.com/sparkdesignsystem/spark-design-system/blob/469233ed9d01df6eebe6b9baebbef4071635555d/html/utilities/generateAriaControls.js